walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,156 @@
+      // 3. The last end time, we can check if we can schedule an item between it and the end of day
+      // First(1) way
+      if (index == 0) {
+        addPossibleTime(START_OF_DAY, startOfCurrentTimeRange, false, durationOfMeeting, possibleTimes);
+      }
+      // Second(2) way
+      if (index +1 < attendeesCannotScheduleHere.size()) {
+        TimeRange currentTimeRangePlusOne = attendeesCannotScheduleHere.get(index + 1);
+        addPossibleTime(endOfCurrentTimeRange, currentTimeRangePlusOne.start(), false, durationOfMeeting, possibleTimes);
+      }
+      //Third(3) way
+      if (index == attendeesCannotScheduleHere.size() - 1) {
+        addPossibleTime(endOfCurrentTimeRange, END_OF_DAY, true, durationOfMeeting, possibleTimes);
+      }
+    }
+    return possibleTimes;
+  }
+
+  /**
+   * Create and check if timerange is appropriate for scheduling requesting time
+   * @param start represents the start time for the time range we want to create
+   * @param end represents the end time for the time range we want to create
+   * @param inclusive this tells us if we would like to include the end time in our time range or not
+   * @param durationOfMeeting the duration of the requested meeting
+   * @param possibleTimes this is a list of all the possible times that we can schedule the meeting
+   */
+  private void addPossibleTime(int start, int end, boolean inclusive, long durationOfMeeting, Collection<TimeRange> possibleTimes) {
+    TimeRange possibleTime = TimeRange.fromStartEnd(start, end, inclusive);
+    if (possibleTime.duration() >= durationOfMeeting) {
+      possibleTimes.add(possibleTime);
+    }
+  }
+
+  /** 
+   * Check for events which attendees are attending and mark those time ranges
+   * @param attendees is a collection of all the attendees for the required meeting
+   * @param events is a collection of all the events that are known
+   * @return a collection of all the time ranges when the {@code attendees} are busy and we cannot schedule meetings there
+  */
+  private List<TimeRange> cannotSheduleMeeting(Collection<String> attendees, Collection<Event> events) {
+    List<TimeRange> attendeesConflictingTimes = new ArrayList<>();
+    for (String currentAttendee: attendees) {
+      for (Event currentEvent: events) {
+        // Check to see if attendee attends the current event
+        if (isAttendeeOfEvent(currentAttendee, currentEvent)) {
+          attendeesConflictingTimes.add(currentEvent.getWhen());
+        }
+      }
+    }
+    return attendeesConflictingTimes;
+  }
+
+  /**
+   * Check if an event has this person as an attendee
+   * @param attendee represents the person we would like to check if they are attending a particular event
+   * @param event represents an event
+   * @return true if {@code attendee} is an attendee of {@code event}
+   */
+  private boolean isAttendeeOfEvent(String attendee, Event event) {
+    return event.getAttendees().contains(attendee) ? true : false;
```

Comment:
.contains should already return a boolean, so I believe you can return it directly: 
`return event.getAttendees().contains(attendee);`